module/friendship/adapter/postgres/repository/user.go

```diff
@@ -39,3 +39,23 @@ func (f UserRepository) GetUserIDsByEmails(ctx context.Context, emails []string)
 
 	return result, nil
 }
+
+func (f UserRepository) GetEmailsByUserIDs(ctx context.Context, userIDs []string) (map[string]string, error) {
+	var users model.Users
+	if err := f.Model(ctx).Table(model.User{}.TableName()).Where("id IN ?", userIDs).Find(&users).Error; err != nil {
+		zResult := make(map[string]string, 0)
```

Comment:
what does the `z` part represent in the variable name ? Please reconsider naming convention